features/cli/create.feature

```diff
@@ -90,10 +90,10 @@ Feature: creating 'apps' with CLI
     #Check git github url
     Given an 8 character random string of type :dns952 is stored into the :appname3 clipboard
     When I run the :new_app client command with:
-      | code         | git://github.com/openshift/ruby-hello-world |
-      | image_stream | openshift/ruby                              |
-      | name         | <%= cb.appname3 %>                          |
-      | env          | MYSQL_ROOT_PASSWORD=test                    |
+      | code         | git@github.com:openshift/ruby-hello-world |
```

Comment:
Format `git@xxx` requires SSH key. Perhaps we can use https for them ?

Reply:
@liangxia the the reason I did that was the comment `#Check git github url`.

Reply:
@liangxia I've updated it now to use `https` instead. PTAL.